osu.Game/Online/Rooms/PlaylistItem.cs

```diff
@@ -30,6 +30,11 @@ public class PlaylistItem : IEquatable<PlaylistItem>
         [JsonProperty("expired")]
         public bool Expired { get; set; }
 
+        [JsonIgnore]
+        public IBindable<bool> Valid => valid;
+
+        private readonly Bindable<bool> valid = new BindableBool(true);
```

Comment:
Not sure if it's just me but I'm not really getting the point of exposing as `IBindable<bool>` if `MarkInvalid()` is going to be a public method anyhow. Feels a little overdone at this point, I guess.

Reply:
I didn't want to allow it to set set to valid again, since I'm not handling that scenario.

Reply:
Would agree with @bdach's sentiments, but /shrug.

Reply:
I'm not against changing it, your call.